### ISSUE TITLE
FormErrorMessage isn't WCAG compliant (Color Contrast)

### ISSUE THREAD

**Issue by lkarvec** [chakra-ui/chakra-ui#10833]
### Description

The color contrast of red.500 is at a 4.1:1 ratio.  WCAG mandates that normal size text be given a 4.5:1 color contrast ratio for those with poor vision.

### Link to Reproduction

https://stackblitz.com/run?file=src%2FApp.tsx&showSidebar=0

### Steps to reproduce

1. put a form error message component on a white page
2. use a color contrast checking tool like ANDI
3. Observe

### Chakra UI Version

Affects all versions.  Tested on 1.x.x, 2.x.x, and 3.x.x

### Browser

Any

### Operating System

- [ ] macOS
- [x] Windows
- [ ] Linux

### Additional Information

https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum has the specific guideline on color contrast.  Note that it is fine against black backgrounds at the moment, but this will not be the average use case of this component.  Yes, I understand users can technically change this color, but the component should be at least WCAG AA compliant out of the box since companies can get sued for disability non-compliance in certain jurisdictions.

**Comment by lkarvec** [comment]
An additional site that might help.  [https://dev.to/finnhvman/which-colors-look-good-on-black-and-white-2pe6](https://dev.to/finnhvman/which-colors-look-good-on-black-and-white-2pe6) has a list of colors that work against both white and black backgrounds.  I recommend swapping the error colors to `#ee0000`.

**Comment by segunadebayo** [comment]
Thanks for this feedback. If you're using Chakra in your project, feel free to update this color in your userland code.

We'll consider this much later as we work towards the next major release.
